Website/js/Debug/dnn.modalpopup.js

```diff
@@ -206,13 +204,12 @@
 
                         var $window = $(window),
                             newHeight,
-                            newWidth,
-                            JQUERY_UI_HEIGHT_SHRINK_OFFSET = 100,
+                            newWidth,                            
                             horizontalPosition = "center",
                             verticalPosition = "center";                            
 
                         if ($modal.data('isMaximized')) {
-                            newHeight = $modal.data('height') + JQUERY_UI_HEIGHT_SHRINK_OFFSET;
```

Comment:
It was adding height to modal in minimized state in fact more height than maximized state need to check  not sure we need that. And as you mentioned why added in https://dnntracker.atlassian.net/browse/DNN-8295